content/10.zk-stack/05.concepts/00.transaction-lifecycle.md

```diff
@@ -159,3 +159,36 @@ These fields are handled by our SDKs.
 
 Each of these transaction types ensures that while ZKsync Era remains closely aligned with Ethereum standards,
 it also optimizes for its Layer 2 specific needs and functionalities.
+
+---
+### Transaction filtering
+The ZK Chain operators can both the transactions initiated from L1 and the ones submitted via API.
```

Comment:
Missing a "filter" in the sentence maybe @dimazhornyk ?

Reply:
Yes, I will add it. And yeah, it makes sense to have it separately, it does belong there more than in the transaction lifecycle